src/main/scala/org/broadinstitute/dsde/firecloud/service/MethodsService.scala

```diff
@@ -0,0 +1,46 @@
+package org.broadinstitute.dsde.firecloud.service
+
+import akka.actor.{Actor, _}
+import com.wordnik.swagger.annotations.{Api, _}
+import org.broadinstitute.dsde.firecloud.MethodsClient
+import org.broadinstitute.dsde.firecloud.model.MethodEntity
+import org.slf4j.LoggerFactory
+import spray.routing.{HttpService, Route}
+
+class MethodsServiceActor extends Actor with MethodsService {
+  def actorRefFactory = context
+  def receive = runRoute(listRoute)
+}
+
+@Api(value = "/methods", description = "Methods Service", produces = "application/json")
+trait MethodsService extends HttpService with FireCloudDirectives {
+
+  private final val ApiPrefix = "methods"
+
+  val listRoute = listMethodsRoute
+
+  lazy val log = LoggerFactory.getLogger(getClass)
+
+  @ApiOperation(
+    value = "list methods",
+    nickname = "listMethods",
+    httpMethod = "GET",
+    response = classOf[MethodEntity],
+    responseContainer = "List",
+    notes = "response is a list of methods from the methods repository")
+  @ApiResponses(Array(
+    new ApiResponse(code = 200, message = "Successful"),
+    new ApiResponse(code = 400, message = "Bad Request"),
```

Comment:
Under what conditions would the 400 and 404 be thrown? See my previous comment - I think we catch those errors and rethrow as a 500.